conduit/plugins/importers/algod/algod_importer.go

```diff
@@ -32,6 +32,11 @@ type algodImporter struct {
 	cancel  context.CancelFunc
 }
 
+func (algodImp *algodImporter) OnComplete(input data.BlockData) error {
```

Comment:
The archival mode importer is still useful for a lot of things. I think the Sync Mode importer should be a new plugin